pages/index.mdx

```diff
@@ -33,7 +33,7 @@ import Link from "next/link"
           or
           </div>
           <div>
-          <h2 className="text-sm md:text-base">Get started with ReadySet open-source </h2>
```

Comment:
Good call, we should eventually remove all mentions to open-source on our docs and website.